lib/gooddata/lcm/actions/synchronize_ldm.rb

```diff
@@ -70,18 +73,20 @@ def sync_segment_ldm(params, segment_info)
           results = []
           client = params.gdc_gd_client
           exclude_fact_rule = params.exclude_fact_rule.to_b
+          include_deprecated = params[:include_deprecated] || false
           from_pid = segment_info[:from]
           from = params.development_client.projects(from_pid) || fail("Invalid 'from' project specified - '#{from_pid}'")
 
           GoodData.logger.info "Creating Blueprint, project: '#{from.title}', PID: #{from_pid}"
-          blueprint = from.blueprint(include_ca: params.include_computed_attributes.to_b)
+          blueprint = from.blueprint(include_ca: params.include_computed_attributes.to_b, include_deprecated: (params[:include_deprecated] || true))
           maql_diff = nil
```

Comment:
why || true?
include_deprecated: (params[:include_deprecated] || true

Reply:
Because it used default = true for calling to model/view
def blueprint(options = {})
      options = { include_ca: true }.merge(options)
      result = client.get("/gdc/projects/#{pid}/model/view", params: { includeDeprecated: true, includeGrain: true, includeCA: options[:include_ca] })